main.go

```diff
@@ -177,12 +147,28 @@ func main() {
 		failf("Failed to set executable permission for gradlew, error: %s", err)
 	}
 
+	// Initialize Android SDK
```

Comment:
This was moved up a bit from line 200 to make `androidSdk` available to the `updateNDK()` method.